### PR TITLE
feat(ol): Add Ordered List component

### PR DESCRIPTION
**Pull request by marceloF5**
A Ordered List component `<Ol />` for list content using `<li />`.

This PR resolves https://github.com/zenorocha/react-email/issues/53.

### CODE REVIEW

File: packages/ol/src/ol.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      ref={forwardedRef}
+      style={{        
+        ...style,
+      }}
+      {...props}
+    />
+  )
+);
+
+Ol.displayName = "Ol";
```
Comment (bukinoshita) [inline packages/ol/src/ol.tsx:20]: This fixes the formatting and the passed style prop.
```suggestion
import * as React from 'react';

type OlElement = React.ElementRef<'ol'>;
type RootProps = React.ComponentPropsWithoutRef<'ol'>;

export interface OlProps extends RootProps {}

export const Ol = React.forwardRef<OlElement, Readonly<OlProps>>(
  ({ style, ...props }, forwardedRef) => (
    <ol ref={forwardedRef} style={style} {...props} />
  ),
);

Ol.displayName = 'Ol';
```

File: packages/ol/src/index.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
```
Comment (bukinoshita) [inline packages/ol/src/index.spec.tsx:1]: Can we rename this file to `ol.spec.tsx` instead of `index.spec.tsx` to match other packages?

**Review by bukinoshita — COMMENTED** [review]
Just some minor cosmetic 💅  things, other than that looks great!

File: packages/ol/package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "dist/**"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "build": "tsup src/index.ts --format esm,cjs --dts --external react",
+    "dev": "tsup src/index.ts --format esm,cjs --dts --external react --watch",
+    "lint": "TIMING=1 eslint src/**/*.ts* --fix",
+    "clean": "rm -rf dist",
+    "test": "jest",
+    "test:watch": "jest --watch"
```
Comment (bukinoshita) [inline packages/ol/package.json:19]: Can you add the format scripts here as well?

```suggestion
    "test:watch": "jest --watch",
    "format:check": "prettier --check \"**/*.{ts,tsx,md}\"",
    "format": "prettier --write \"**/*.{ts,tsx,md}\""
```

**Comment by zenorocha** [comment]
This is SO cool! Thanks again @marceloF5

File: packages/ol/package.json (hunk 3 above)
Comment (marceloF5) [reply packages/ol/package.json]: Done!

File: packages/ol/src/index.spec.tsx (hunk 2 above)
Comment (marceloF5) [reply packages/ol/src/index.spec.tsx]: Done!

File: packages/ol/src/ol.tsx (hunk 1 above)
Comment (marceloF5) [reply packages/ol/src/ol.tsx]: Done!

File: packages/ol/src/ol.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+export interface OlProps extends RootProps {}
+
+export const Ol = React.forwardRef<OlElement, Readonly<OlProps>>(
+  ({ style, ...props }, forwardedRef) => (
+    <ol
+      ref={forwardedRef}
+      style={{
+        ...style,
+      }}
```
Comment (bukinoshita) [inline packages/ol/src/ol.tsx:14]: No need to destruct here.

```suggestion
      style={style}
```

File: packages/ol/src/ol.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+describe('render', () => {
+  beforeEach(() => {
+    jest.restoreAllMocks();
+    jest.resetModules();
+  });
+
+  it('renders the <Ol> component', () => {
+    const actualOutput = render(
+      <Ol>
+        <li>foo</li> <li>bar</li>
```
Comment (bukinoshita) [inline packages/ol/src/ol.spec.tsx:14]:
```suggestion
        <li>foo</li>
        <li>bar</li>
```

File: packages/ol/src/ol.tsx (hunk 4 above)
Comment (marceloF5) [reply packages/ol/src/ol.tsx]: Fixed!

File: packages/ol/src/ol.spec.tsx (hunk 5 above)
Comment (marceloF5) [reply packages/ol/src/ol.spec.tsx]: Take advantage to fix `<Ul />`

**Comment by bukinoshita** [comment]
Nice! Looks great @marceloF5, thanks! 🎉

**Comment by marceloF5** [comment]
@bukinoshita  I fixed unnecessary react imports in `<ol />` and `<ul />` .specs files. Could you check, please?

**Comment by bukinoshita** [comment]
@zenorocha can you merge this PR? Looks like the contributors PRs not triggering CI, but I checked locally and it works fine. I already published to npm. So the remaining part is to merge.

**Comment by bukinoshita** [comment]
Never mind, I was able to change the configuration on CircleCI — It's now running on forked repositories as well.

@marceloF5 Can you run `yarn format`? The CI is complaining about it, I think that's the only thing left.

**Comment by zenorocha** [comment]
🎉🎉🎉

**Comment by bukinoshita** [comment]
I think the tests were failing on this one

**Pull request merged**